core/src/main/java/de/muenchen/allg/itd51/wollmux/former/Broadcast.java

```diff
@@ -23,15 +23,15 @@
 package de.muenchen.allg.itd51.wollmux.former;
 
 /**
- * Interface für Nachrichten auf dem globalen Broadcast-Kanal. Siehe auch
+ * Interface for messages on the global broadcast channel. See also
```

Comment:
Hmm - a literal translation is not ideal here (as this is an interface description).

Its an implementation of https://en.wikipedia.org/wiki/Publish-subscribe_pattern - so perhaps add a link in the line below, and tweak the mission statement to 'Interface for sending a message on the global pubsub channel' (or somesuch)?